agr_mvs/tienda.py

```diff
@@ -1,18 +1,26 @@
 # Clase Tienda (entidad que representa cada Tienda de nuestra aplicación)
 
-from src.empresa import Empresa
-from src.empleado import Empleado
-from src.enums.enum_turno import Turno
+from agr_mvs.empresa import Empresa
+from agr_mvs.Turno import Turno
 
 class Tienda:
+    # Nombre de la Tienda, Empresa para establecer el dato
+    id: int                         # Identificador de la tienda
     nombre: str                     # Nombre de la tienda
     empresa: Empresa                # Empresa a la que pertenece la tienda
```

Comment:
¿necesitas id, nombre y empresa? ¿No será suficiente con que pongas un nombre que sea único?
Recordad que tenéis que hacer un issue también para esto, o reabrir los del objetivo 2.

Reply:
He dejado finalmente el nombre y he eliminado la clase empresa  y el id innecesario